source/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp

```diff
@@ -1,20 +1,49 @@
 /*
-This example shows how to utilize the result of Eye-in-Hand calibration to transform (picking) point
-coordinates from the camera frame to the robot base frame. The YAML files for this sample can be found under the main
-instructions for Zivid samples.
+This example shows how to utilize the result of Eye-in-Hand calibration to transform either a (picking) point
```

Comment:
Check if description changes have to be made also in the main Readme.md

Reply:
I've already changed the main ReadMe from "Transform point from camera frame..." to "Transform single data point or entire point cloud..."